packages/create-app/templates/default-app/packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -184,7 +184,7 @@ const ApiEntityPage = ({ entity }: { entity: Entity }) => (
 export const EntityPage = () => {
   const { entity } = useEntity();
 
-  switch (entity?.kind?.toLowerCase()) {
+  switch (entity?.kind?.toLocaleLowerCase("en-US")) {
```

Comment:
Should be single quotes here instead otherwise it will fail linting of created projects I believe.